vscode/open/command.py

```diff
@@ -98,6 +100,33 @@ def command(shell, args: list):
 
         # get info about project
         project = DTProject(parsed.workdir)
+        
+        try:
+            devcontainer_configuration : DevContainerConfiguration = project.devcontainers[parsed.devcontainer_config]
+        except KeyError as e:
+            dtslogger.error(f"Could not find the {e} configuration in 'devcontainers.yaml'")
+            return
```

Comment:
If a devcontainer is not defined we don't give an error, we simply run VSCode without devcontainer. We want this to be backward compatible with the MOOC and others LXs that do not have and do not need devcontainers.

Reply:
Do you mean simply opening vscode in the current folder or launching the lx with `dts code workbench`?

Reply:
`dts code workbench` uses internally the vscode open command, we want to retain the ability to use it to open an instance of vscode.

Also modify the `--devcontainer` command to behave as the `-M` flag in `dts devel run`: if we only pass the flag `--devcontainer` it uses the `default` one and if we pass also a value `--devcontainer=mydevcontainer` we want to open `mydevcontainer`